### DISCUSSION TITLE
Where is the documentation about picture/image resizing?

### DISCUSSION THREAD

**Discussion by meotimdihia** [seaweedfs/seaweedfs#2732] [Q&A]
Where is the documentation about picture resizing? I can't find anything about this feature.
I have so many questions about this feature:
What happens to images after resizing? Did Seaweedfs store it in memory?
Do we need to security this feature when someone exploit it by change params `?height=[random]&width=[random]` ?

**Comment by flomo29** [comment]
There are many information on the volume server api page. The resizing feature ist when you read data with HTTP GET command. the volume server can resize the picture when reading on the fly, so when you need different pictures sizes, you can store the picture only ones and read it multiple times with different size parameters, to get different pictures. try it out!

**Reply by meotimdihia** [reply]
My question is different. I tried to use this feature in production servers and the server is full of memory after minutes.

**Reply by chrislusf** [reply]
This resizing feature uses code written in Go, which does need a lot of memory. This feature was deleted but added back due to requests.

**Reply by meotimdihia** [reply]
> This resizing feature uses code written in Go, which does need a lot of memory. This feature was deleted but added back due to requests.

Thanks for the explanation. Please keep this feature, it is still useful. Since I know it uses memory, and now I am using it for resizing huge images only. 😀 It is even better than if we could save resized images to disk or limit sizes for security.
